tests/integration/memcached/test_socket.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*DESCRIPTION
+The agent should report instance metrics when multiple servers are
+added at once
+*/
+
+/*SKIPIF
+<?php require('skipif.inc'); ?>
+*/
+
+/*INI
+*/
+
+/*EXPECT_METRICS_EXIST
+Datastore/instance/Memcached/__HOST__/my_socket, 1
+*/
+
+/*EXPECT_ERROR_EVENTS null */
+
+require_once(realpath (dirname ( __FILE__ )) . '/../../include/helpers.php');
+require_once(realpath (dirname ( __FILE__ )) . '/../../include/tap.php');
+require_once(realpath (dirname ( __FILE__ )) . '/memcache.inc');
+
+$memcached = new Memcached();
+$memcached->addServer("my_socket", 0);
```

Comment:
Could this be tested with an actual path to the socket? I.e. have the first argument include `/` (forward slash) directory separator?

Reply:
added in [8ec0268](https://github.com/newrelic/newrelic-php-agent/pull/958/commits/8ec02684f0bb8726ae111d78d37e0dd492d662c5)